src/main/java/org/spongepowered/api/data/Queries.java

```diff
@@ -42,8 +42,8 @@
     public static final DataQuery WEIGHTED_SERIALIZABLE = of("DataSerializable");
     public static final DataQuery WEIGHTED_SERIALIZABLE_WEIGHT = of("Weight");
 
-    // Item Enchantment
-    public static final DataQuery ENCHANTMENT_ID = of("Enchantment");
+    // Item EnchantmentType
+    public static final DataQuery ENCHANTMENT_ID = of("EnchantmentType");
```

Comment:
This was done automatically by IntelliJ. Should I keep I revert this change to not cause compatibility issues?

Reply:
Anyone have input here?

Reply:
undo

Reply:
Will do. 👍